test/org.eclipse.elk.alg.force.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: ELK Force Tests
+Bundle-SymbolicName: org.eclipse.elk.alg.force.test;singleton:=true
+Bundle-Version: 0.8.0.qualifier
+Automatic-Module-Name: org.eclipse.elk.alg.force.test
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: Eclipse Modeling Project
+Require-Bundle: com.google.guava;bundle-version="15.0.0",
```

Comment:
Why guava `15.0.0`?

Reply:
Simply copied a different plugin as template. 

We could make the version consistent across all plugins though :).

Reply:
@soerendomroes I ran a grep on the repo and it turns out that whenever the version is set, it is set to 15 (with the exception of two test plugins, which use 18). I'm still open for an update though.

Reply:
I'll create an issue for bumping the Guava version.